### PR TITLE
🐛 Fix patch/add/remove project node

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
- 🐛 From discussion with @GitHK `update_dynamic_service_networks_in_project` should be called on creation, deletion and rename of a project node
- 🐛 On patching of a project/project node sync data with director-v2

NOTE: @odeimaiz this should fix Carina issue in Master

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py [hunk 1]
```diff
@@ -333,6 +333,7 @@ async def update_projects_networks(
         RabbitMQClient, Depends(get_rabbitmq_client_from_request)
     ],
 ) -> None:
+    # NOTE: This needs to be called to update networks only when adding, removing, or renaming a node.
```
Comment (GitHK) [inline services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py:336]: I would put this as a comment inside the function not here. Maybe as a doctoring

**Review by GitHK — APPROVED** [review]
Thanks

**Review by odeimaiz — APPROVED** [review]
merci

**Review by sanderegg — APPROVED** [review]
thanks

**Review by pcrespov — APPROVED** [review]

**Pull request merged**